content/docs/components-and-props.md

```diff
@@ -173,11 +172,11 @@ function Avatar(props) {
 }
 ```
 
-The `Avatar` doesn't need to know that it is being rendered inside a `Comment`. This is why we have given its prop a more generic name: `user` rather than `author`.
+El `Avatar` no necesita saber que está siendo renderizado dentro de un `Comment`. Esto es por lo que le dimos a la propiedad un nombre más genérico: `user` en vez de `author`.
```

Comment:
I suggest changing:
> Esto es por lo que le dimos a la propiedad un ...

by 
>Por este motivo le dimos a su propiedad un...

Note how they say "its prop", not "the prop", so I changed `a` by `su`.

Reply:
> Esto es por lo que le dimos a su propiedad...
What do you think?

Reply:
I think that `Por este motivo ...` better describes the situation here.

Reply:
I agree and also sounds better that way. But for me `Esto es por lo que` is closer to the original` what do you think?